content/applications/finance/expenses.rst

```diff
@@ -4,231 +4,549 @@
+  filed for from the drop-down menu. If there is only one company, this field will be automatically
+  populated.
+- :guilabel:`Notes...` : If any notes are needed in order to clarify the expense, enter them in the
+  notes field.
+
+ Once all the fields have been filled out, click :guilabel:`Save`.
+
+Attach a receipt
+~~~~~~~~~~~~~~~~
+
+After the expense is saved, the next step is to attach a receipt. A new :guilabel:`Attach Receipt`
+button appears after the entry is saved, beneath the former :guilabel:`Save` button (which turns
+into an :guilabel:`Edit` button).
+
+.. image:: expenses/save_receipt.png
+   :align: center
+   :alt: Attach a receipt after saving the record.
+
+Click the new :guilabel:`Attach Receipt` button, and a file explorer appears. Navigate to the
+receipt to be attached, and click :guilabel:`Open`. A new :guilabel:`Receipts` smart button appears
+at the top, and the receipt appears in the chatter. More than one receipt can be attached to an
+individual expense, if needed. The number of receipts attached to the expense will be noted on the
+smart button.
+
+.. image:: expenses/receipt_smartbutton.png
+   :align: center
+   :alt: Attach a receipt after saving the record.
+
+Automatically create new expenses from an email
+-----------------------------------------------
+
+Instead of individually creating each expense in the :guilabel:`Expenses` app, expenses can be
+automatically created by sending an email to an email alias.
+
+To do so, first, an email alias needs to be configured. Go to :menuselection:`Expenses app -->
+Configuration --> Settings`. Ensure :guilabel:`Incoming Emails` is checked off.
+
+.. image:: expenses/email.png
+   :align: center
+   :alt: Create the domain alias by clicking the link.
+
+.. note::
+   If the domain alias needs to be set up, :guilabel:`Setup your domain alias` will appear beneath
+   the incoming emails check box instead of the email address field. If the domain alias needs to be
+   setup, refer to this documentation: :doc:`/administration/maintain/domain_names`. Once the domain
+   alias is configured, the email address field will be visible beneath the incoming emails section.
+
+Next, enter the email address to be used in the email field, then click :guilabel:`Save`. Now that
+the email address has been entered, emails can be sent to create new expenses without having to be
+in the Odoo database.
+
+To submit an expense via email, create a new email and enter the product's internal reference code
+(if available) and the amount of the expense in the email subject. Next, attach the receipt to the
+email. Odoo creates the expense by taking the information in the email subject and combining it with
+the receipt.
+
+To check an expense product's :guilabel:`Internal Reference`, go to :menuselection:`Expenses app -->
+Configuration --> Expense Products`. If an internal reference is listed on the product, it is
+visible in this view.
```

Comment:
Pretty sure it would be good to express how the internal reference is visible here @StraubCreative is this valid 👇?
```suggestion
visible in this view as :guilabel:`(Ref###)`.
```